torch/_export/serde/schema.py

```diff
@@ -10,10 +10,28 @@ class _Union:
     @classmethod
     def create(cls, **kwargs):
         assert len(kwargs) == 1
-        return cls(**{**{field.name: None for field in fields(cls)}, **kwargs})
+        obj = cls(**{**{f.name: None for f in fields(cls)}, **kwargs})
+        return obj
+
+    @classmethod
+    def fields(cls):
+        fields_name = [(f.name, str, field(default=f.name)) for f in fields(cls)]
+        return make_dataclass("FieldDataClass", fields_name)()
 
     def __post_init__(self):
-        assert sum(1 for field in fields(self) if getattr(self, field.name) is not None) == 1
+        assert sum(1 for f in fields(self) if getattr(self, f.name) is not None) == 1
+
+    @property
+    def value(self):
+        val = [getattr(self, f.name) for f in fields(self) if getattr(self, f.name) is not None]
```

Comment:
`__post_init__` should guarantee this invariant, so you should be able to just do `next(...)` here, right?